frontend/src/general/userForm/FormConsole.jsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { Button, Container, TextField } from "@material-ui/core";
+import PropTypes from "prop-types";
+
+export default class FormConsole extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: "",
+      password: ""
+    };
+    this.onSubmit = this.onSubmit.bind(this);
+    this.setUsername = this.setUsername.bind(this);
+    this.setPassword = this.setPassword.bind(this);
+  }
+
+  onSubmit(event) {
+    const { handleSubmit } = this.props;
+    const { username, password } = this.state;
+    event.preventDefault();
+    handleSubmit(username, password);
+  }
+
+  setUsername(username) {
+    this.setState({
+      username
+    });
+  }
+
+  setPassword(password) {
+    this.setState({
+      password
+    });
+  }
+
+  render() {
+    const { buttonName, isValidate, isCreateUser } = this.props;
+    const { password } = this.state;
```

Comment:
uhh rationale behind getting just the password here? & feeding its value to the associated textfield? why not the same for username? why do this at all in the first place when they both start as empty strings?

Reply:
If you look at lines 65-69, I need the password state for the length check to make sure it's between 5 to 20 characters. I don't need to pass username because I don't need to reference the state in any of the component props.

Reply:
ahh i see! but in that case why is `value={password}` still necessary? i tried commenting it out on my end and it didn't change the behaviour at all

Reply:
I think that was leftover from testing things out and I didn't remove it, if it didn't change the behavior I'll remove it!